doc/user_guide/interactions/parameters.rst

```diff
@@ -296,7 +318,7 @@ selection:
         return alt.Chart(cars).mark_rect().encode(
             x="Cylinders:O",
             y="Origin:N",
-            color=alt.condition(selector, 'count()', alt.value('lightgray'))
+            color=alt.when(brush).then("count()").otherwise(alt.value("lightgray")),
         ).properties(
```

Comment:
None of the charts in this section rendered for me locally for some reason. Just a note to double check in the final review

Reply:
Interesting, I'm just rebuilding the docs now to see if I can spot the issue

Reply:
Fixed in [fix: Use `selector` argument in `make_example`](https://github.com/vega/altair/pull/3544/commits/fb42da7de606ec03b8d58773479374492f6e93be)